06-r-and-sql.Rmd

```diff
@@ -0,0 +1,144 @@
+---
+title: "SQL databases and R"
+output:
+  html_document:
+    toc: true
+    theme: united
+---
+
+
+Learning goals
+------
+Students should finish the lesson with the following skills.
+
+* Students will be able to connect to an existing database using R
+* Query an existing database through R using queries constructed with variables
+* Create a database and populate it from R
+* Understand that scripted database interactions enhance reproducibility of analysis
+
+
+Introduction
+---------
+
+Interacting with databases through scripted languages can offer advantages over querying databases via a GUI interface.  A GUI interfaces for your database is easier to use and allows the quick viewing of adhoc queries.  Querying a database with a programatic intreface (in this case R, but it could be any language) are slightly more complicated.  However the trade-off is that data manipulations are preserved in the code.  Aggregations, summaries and other database operations are preserved.  Therefore those pre-analysis data manipulation steps are not lost.
```

Comment:
are not lost --> are not lost and can be reproduced later? or something along these lines.